### PR TITLE
add text normlization

### PR DESCRIPTION
**Pull request by zh794390558**
### PR types
Others

### PR changes
Docs 

### Describe
text norm docs

### CODE REVIEW

File: doc/src/tts_text_front_end.md [hunk 1]
```diff
@@ -13,6 +13,37 @@ There are various libraries including some of the most popular ones like NLTK, S
 
 ## Text Normalization(文本正则)
 
+The **basic preprocessing steps** that occur in English NLP, including data cleaning,stemming/lemmatization, tokenization and stop words. **not all of these steps are necessary for Chinese text data!**
```
Comment (iclementine) [inline doc/src/tts_text_front_end.md]: stop-words removal

File: doc/src/tts_text_front_end.md [hunk 2]
```diff
@@ -13,6 +13,37 @@ There are various libraries including some of the most popular ones like NLTK, S
 
 ## Text Normalization(文本正则)
 
+The **basic preprocessing steps** that occur in English NLP, including data cleaning,stemming/lemmatization, tokenization and stop words. **not all of these steps are necessary for Chinese text data!**  
+
+### Lexicon Normalization
```
Comment (iclementine) [inline doc/src/tts_text_front_end.md:18]: 这里确切指的是什么？ lemmatization?

File: doc/src/tts_text_front_end.md [hunk 3]
```diff
@@ -13,6 +13,37 @@ There are various libraries including some of the most popular ones like NLTK, S
...
+
+These “tags” label the part of speech. There are 24 part of speech tags and 4 proper name category labels in the `**jieba**` package’s existing dictionary.
+
+<img src="../images/jieba_tags.png" width=650>
+
+### Stop Words
+
+In NLP, **stop words are “meaningless” words** that make the data too noisy or ambiguous. 
+
+Instead of manually removing them, could import the `**stopwordsiso**` package for a full list of Chinese stop words.More information can be found [here](https://pypi.org/project/stopwordsiso/).And with this, we can easily create code to filter out any stop words in large text data.
```
Comment (iclementine) [inline doc/src/tts_text_front_end.md]: We could? or You could?
add space after period(`.`)

File: doc/src/tts_text_front_end.md (hunk 1 above)
Comment (iclementine) [reply doc/src/tts_text_front_end.md]: add space after comma(`,`)

**Review by iclementine — COMMENTED** [review]
Typos need to be fixed.

File: doc/src/tts_text_front_end.md (hunk 2 above)
Comment (zh794390558) [reply doc/src/tts_text_front_end.md:18]: 应该指的是英文的词根吧。

**Review by iclementine — APPROVED** [review]
LGTM

**Pull request merged**